common/src/main/java/org/keycloak/common/util/ObjectUtil.java

```diff
@@ -76,4 +79,28 @@ public static boolean isBlank(final CharSequence cs) {
         }
         return true;
     }
+
+    /**
+     * Calling:
+     * <pre>joinValuesWithLogicalCondition("or", Arrays.asList("foo", "bar", "baz", "caz" ))</pre>
+     * will return "foo, bar, baz or caz"
+     *
+     * @param conditionText condition
+     * @param values values to be joined with the condition at the end
+     * @return see above
+     */
+    public static String joinValuesWithLogicalCondition(String conditionText, Collection<String> values) {
```

Comment:
More like a string utility, no? See org.keycloak.utils.StringUtil. Perhaps we should move this class to core too.

Reply:
@pedroigor Good point, I've moved from `ObjectUtil` to `StringUtil` . I've used separate commit, so you can re-review just this simple change.